src/Command/ImportScorerCommand.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\Command;
+
+use App\Command\Adapter\AdapterInterface;
+use App\Entity\Scorer;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class ImportScorerCommand extends Command
+{
+    const OPTION_OVERWRITE = '--overwrite';
+    const OPTION_FORCE = '--force';
+    const ARGUMENT_PATH = 'path';
+
+    protected $encoder;
```

Comment:
As PHP does not support property types in version 7.2 you might want to add PHPDocs here with the type for each property.